index.html

```diff
@@ -9,81 +9,85 @@
     <meta name="description" content="The OpenTF Foundation. Supporting an impartial, open, and community-driven Terraform." />
 
     <link rel="canonical" href="/" />
-    <link rel="stylesheet" type="text/css" href="/style.css">
+    <link rel="stylesheet" type="text/css" href="style.css">
   </head>
   <body>
     <div class="container mx-auto">
       <h1>The OpenTF Manifesto</h1>
 
       <div>
         <p>
-          On August 10th, 2023 a single decision was made which took the DevOps
-          world by surprise. A previously benevolent guardian of an open source
-          project called Terraform decided to relicense their project to new
-          vague terms which effectively put any player in the ecosystem at their
-          mercy.
+          Terraform was open sourced in 2014 under an MPL license, and over the next ~9 
+          years, it built up a community that included thousands of users, contributors, 
+          customers, certified practictioners, and vendors, and an ecosystem of open source 
+          modules, plugins, libraries, and extensions. Then, on August 10th, 2023, with no 
+          advance notice, and no chance for the community to have any input, HashiCorp 
+          switched Terraform to a non open source BSL license. This change threatens the 
+          entire Terraform community and ecosystem. 
         </p>
 
         <p>
-          And with that, millions of lines of Terraform code became legacy
-          overnight. Tens of thousands of businesses, ranging from one-man shops
-          to the Fortune 500 woke up to a new reality where the underpinnings of
-          their cloud presence suddenly became a legal risk. Thousands of people,
-          often deep in love with Terraform, found themselves in a position
-          where their hard-earned skills are in danger of becoming irrelevant.
+          Overnight, tens of thousands of businesses, ranging from one-man shops
+          to the Fortune 500, woke up to a new reality where the underpinnings of
+          their infrastructure suddenly became a potential legal risk. The legal terms in
+          the BSL license are vague, and now every company, every vendor, and every 
+          developer who is using Terraform has to wonder whether what they are doing could
+          be construed as competitive with HashiCorp. And even if you might be in the clear now,
+          how can you build confidence that your usage won't violate the license terms in the future?
```

Comment:
This is important. Worth a bold/stress?

Reply:
Good thought. I added a powerful, bold header: https://github.com/opentffoundation/manifesto/pull/20/commits/910ae0e2a695b976981ea34577483186d0050742

"Our concern: the BSL license is a poison pill for Terraform."